compatibility/common/src/test/java/org/projectnessie/tools/compatibility/internal/TestNessieCompatibilityExtensions.java

```diff
@@ -100,11 +117,14 @@ void tooManyExtensions() {
 
   @Test
   void olderClients() {
-    EngineTestKit.engine(MultiEnvTestEngine.ENGINE_ID)
-        .configurationParameter(
-            "nessie.versions", NEW_STORAGE_MODEL_WITH_COMPAT_TESTING.toString() + ",current")
-        .selectors(selectClass(OldClientsSample.class))
-        .execute();
```

Comment:
these tests were not checking the test events for failures... they relied on the assertions below verifying expected state on the static fields of the class under test
(not sure if this is a good pattern?
maybe other tests in this file should use `assertNoFailedTestEvents` as well?)

Reply:
Checking test events seems reasonable to me.

Reply:
so we should add it for all other tests in this file as well?

Reply:
Yes, I think it makes sense for all tests. The way we run them, does not automatically propagate JUnit5 failures to the outer test.

Reply:
done